src/components/common/Directors.vue

```diff
@@ -1214,32 +1172,23 @@ export default class Directors extends Mixins(CommonMixin, DateMixin, DirectorMi
   }
 
   /**
-   * Local helper to watch changes to the address data in BaseAddress component, and to update
-   * our inProgressAddress holder. To be used when we want to save the data.
+   * Called when BaseAddress component has an updated delivery address.
+   * To be used when we want to save the data.
    * @param val The new value.
    */
-  private updateBaseAddress (val: AddressIF): void {
-    this.inProgressAddress = val
+  private updateDeliveryAddress (val: AddressIF): void {
+    this.inProgressDelivAddress = val
   }
 
   /**
-   * Local helper to watch changes to the mailing address data in BaseAddress component, and to update
-   * our inProgressMailAddress holder. To be used when we want to save the data.
+   * called when BaseAddress component has an updated mailing address.
+   * To be used when we want to save the data.
    * @param val The new value.
    */
   private updateMailingAddress (val: AddressIF): void {
     this.inProgressMailAddress = val
   }
 
-  /**
-   * Function called externally to set all directors from existing filing.
-   * FUTURE: change this to a prop
-   * @param directors The list of directors to set.
-   */
-  public setAllDirectors (directors: DirectorIF[]): void {
```

Comment:
Again, I refactore the interface between parent and child (ie, API) and this method was not needed.